### PR TITLE
feat(mysql8): mysql8 compatibility + package upgrades

### PR DESCRIPTION
**Pull request by kschelonka**
This snowballed into more upgrades, but includes the knex+mysql2 package upgrades for mysql8 compatibility. 

See also: https://github.com/Pocket/curation-tools-data-sync/pull/635

Changes:
* A ton of formatting updates (unfortunately) -- I think the linting rules changed
* Library namespace upgrades for terraform imports, to upgrade to the latest version of cdktf
* the mysql2/knex3 upgrades
* other library upgrades to just get us up to date
* a small post-install script to make it easy to install all packages with one command from the root directory

### CODE REVIEW

File: bin/db.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const connection = {
 };
 
 export const dbClient = knex({
-  client: 'mysql',
+  client: 'mysql2',
```
Comment (kschelonka) [inline bin/db.ts:13]: the big change 💪

File: buildspec.yml [hunk 2]
```diff
@@ -34,6 +34,8 @@ phases:
     commands:
       - . /home/circleci/.codebuild_shims_wrapper.sh
       - echo $CODEBUILD_WEBHOOK_HEAD_REF
+      - nvm install --no-progress
```
Comment (kschelonka) [inline buildspec.yml:37]: install and use the version of node in .nvmrc

**Review by bassrock — APPROVED** [review]
LGTM so much house keeping

**Comment by kschelonka** [comment]
I ran `npm run lint-fix` so I'm going to go ahead and merge

**Pull request merged**